src/engine/tools/engine-suite/src/engine_router/cmds/update.py

```diff
@@ -0,0 +1,56 @@
+import os
+from google.protobuf.json_format import ParseDict
+from shared.default_settings import Constants
+from shared.dumpers import dict_to_yml
+
+from api_communication.client import APIClient
+import api_communication.proto.router_pb2 as erouter
+import api_communication.proto.engine_pb2 as engine
+
+
+def run(args):
+
+    # Get the params
+    api_socket: str = args['api_socket']
+    route: str = args['route']
+    valid_update = False
+
+    # Create API client
+    client = APIClient(api_socket)
+
+    # Create the request
+    request = erouter.RoutePatchPriority_Request()
+    request.name = route
+
+    # Check if the priority must be updated and if it is valid
+    if args['priority'] is not None:
```

Comment:
I think this should be a mandatory parameter like this when adding a route

Reply:
Not really, we have yet to improve the route updates, allowing to change the filter and or policy. This would imply to modify this code again by changing the already defined arguments and the user interface.